src/components/Editor/Invitees/InviteesListSearch.vue

```diff
@@ -186,7 +186,7 @@ export default {
 						name = email
 					}
 
-					if (this.alreadyInvitedEmails.includes(email)) {
+					if (this.alreadyInvitedEmails.includes(email) && email) {
```

Comment:
but now it skips the attendee if it has an email, right?

Reply:
No. If user has email, he/she only skip when this.alreadyInvitedEmails.includes(email) === true, and skip all users don't have email.
I updated the condition check order in latest commit, maybe this looks more intuitive.
In country like China, a lot of people don't have email, so updated this  :)